index.d.ts

```diff
@@ -0,0 +1,55 @@
+type RecordObject = import('cql-execution').RecordObject;
+type PatientObject = import('cql-execution').PatientObject;
+type DataProvider = import('cql-execution').DataProvider;
+type RetrieveDetails = import('cql-execution').RetrieveDetails;
+type Code = import('cql-execution').Code;
+type Date = import('cql-execution').Date;
```

Comment:
The relevant functions actually return a `DateTime`, not a `Date`.
```suggestion
type DateTime = import('cql-execution').DateTime;
```